pkg/csi/service/vanilla/controller_test.go

```diff
@@ -401,6 +420,76 @@ func TestCreateVolumeWithStoragePolicy(t *testing.T) {
 	}
 }
 
+//For this test, when the testbed has multiple shared datastores
```

Comment:
Does this require a vCenter Server? Or this is a unit test and there is no vSphere dependency? I still did not understand how this test works.

Reply:
All the tests in "controller_test.go" can be running with two modes. unit test and integration test. When it is running as integration test, it need to specify a VC IP.

Reply:
In case of "unit test" mode, how are we making sure that we have some datastores that do not have necessary priv?